kernelci/build.py

```diff
@@ -347,7 +347,10 @@ def push_tarball(config, kdir, storage, api, token):
 
 
 def _download_file(url, dest_filename, chunk_size=1024):
-    resp = requests.get(url, stream=True)
+    headers = {
+        'User-Agent': 'kernelci-core build.py agent 1.0'
```

Comment:
We should probably be using the Python package name and version, i.e. `kernelci 1.1`.  At the moment that's all defined in `setup.py` but we could have the version number in `kernelci/__init__.py` and import it here.